external-libs/README.md

```diff
@@ -0,0 +1,2 @@
+This directory is used to copy code from other projects in-tree. This
```

Comment:
In other CAP* project I've seen using `/third-party/` to have external copied dependencies, would you consider switching for consistency?